config/main.py

```diff
@@ -1420,6 +1421,11 @@ def apply_patch(ctx, patch_file_path, format, dry_run, ignore_non_yang_tables, i
 
 @config.command()
 @click.argument('target-file-path', type=str, required=True)
+@click.option('-s', '--scope',
+              help='Specify the namespace for Multi-ASIC environments. For Single-ASIC environments, \
+                specifying the namespace is not required.',
+              required=True if multi_asic.is_multi_asic() else False,
```

Comment:
>True [](http://example.com/codeflow?start=23&length=4)

Is there a valid requirement that we need to handle ASIC scope? Is it whole device `config replace` good enough?